content/concepts/introduction/protocols/identify.md

```diff
@@ -4,4 +4,39 @@ description: "A way for nodes in libp2p to query and inform other nodes about th
 weight: 23
 ---
 
-Coming soon!
+The libp2p identify protocol is a way for nodes to query and inform other nodes
+about their metadata, such as their public key and the protocols they support.
```

Comment:
```suggestion
The libp2p Identify protocol is a way for nodes to query and inform other nodes
about their metadata, such as their public key and the protocols they support.
```

Reply:
I think we should have a list here of what Identify does, namely:
* the user agent (explain what that is)
* informs the peer of their public IP address
* ships the list of the multiaddresses we're listening on
* ships the list of protocols we support

(This probably needs to be polished a bit to sound nicer)

Reply:
That sounds good